pkg/productinfo/google/productinfo_google.go

```diff
@@ -236,32 +236,30 @@ func (g *GceInfoer) GetProducts(ctx context.Context, regionId string) ([]product
 	if err != nil {
 		return nil, err
 	}
-	for _, zone := range zones {
-		err = g.computeSvc.MachineTypes.List(g.projectId, zone).Pages(context.TODO(), func(allMts *compute.MachineTypeList) error {
-			for _, mt := range allMts.Items {
-				if _, ok := vmsMap[mt.Name]; !ok {
-					switch {
-					case mt.GuestCpus < 1:
-						// minimum 1 Gbps network performance for each virtual machine
-						ntwPerf = 1
-					case mt.GuestCpus > 8:
-						// theoretical maximum of 16 Gbps for each virtual machine
-						ntwPerf = 16
-					default:
-						// each vCPU has a 2 Gbps egress cap for peak performance
-						ntwPerf = uint(mt.GuestCpus * 2)
-					}
-					vmsMap[mt.Name] = productinfo.VmInfo{
-						Type:    mt.Name,
-						Cpus:    float64(mt.GuestCpus),
-						Mem:     float64(mt.MemoryMb) / 1024,
-						NtwPerf: fmt.Sprintf("%d Gbit/s", ntwPerf),
-					}
+	err = g.computeSvc.MachineTypes.List(g.projectId, zones[0]).Pages(context.TODO(), func(allMts *compute.MachineTypeList) error {
```

Comment:
is it intentional that the `zone` cycle was removed?

Reply:
yes, this is by intention as confirmed by @prekoa